react/StoreWrapper.js

```diff
@@ -146,13 +148,14 @@ const StoreWrapper = ({ children }) => {
           .filter(Boolean)
           .filter(meta => meta.content && meta.content.length > 0)}
         script={[
-          supportsServiceWorker && {
-            type: 'text/javascript',
-            src: `${rootPath}/pwa/workers/register.js${queryMatch}&scope=${encodeURIComponent(
-              rootPath
-            )}`,
-            defer: true,
-          },
+          supportsServiceWorker &&
+            serviceWorker && {
+              type: 'text/javascript',
+              src: `${rootPath}/pwa/workers/register.js${queryMatch}&scope=${encodeURIComponent(
+                rootPath
+              )}`,
+              defer: true,
+            },
```

Comment:
I don't think this will work for stores/users that already have our service worked installed in their computer. we might want instead to have this "register" file with the `__disableSW=true` query string, so our service worker will effectively be unregistered from the store and enable 3rd party service workers.

I think we will need to open a PR in pwa-graphql to change [the unregister logic](https://github.com/vtex/pwa-graphql/blob/master/node/workers/ioServiceWorker.js#L578-L584) to only call the `unregister` method on our own SW, instead of all the service workers installed

Reply:
Yes, working on it!

Reply:
Updated the PR description and added a link to the https://github.com/vtex/pwa-graphql/pull/87